### ISSUE TITLE
Second Release

### ISSUE THREAD

**Issue by danielwiktorczyk** [The-GameDJs/CostumeCurse#113]
The idea is to complete all the IMPORTANT STUFF below before we start mid-Winter 2021. 

**Important Stuff**
- [x] Enemy Special Abilities
- [ ] Balancing what we already have
- [ ] Boss Fight Abilities (Spawning + Memorized Dodge)
- [ ] Bug Fixes
- [ ] Remake Map
- [ ] Mandatory Cut Scenes for Important Dialogue (Witch + Frog + Boss)
- [ ] Music
- [ ] Sfx
- [ ] Get our story straight (especially past the village)

**Stretch Goals**
- [ ] Enemy Dialogue?
- [ ] Shaders
- [ ] Mini MINI quests (search + find, kill THAT enemy)

**FUTURE probably not in this release**
- [ ] Items (+ rethinking the gameplay)
